addon/components/tf-input/input-description/template.hbs

```diff
@@ -0,0 +1 @@
+{{text}}
```

Comment:
`input-description` is supposed to be "private", so it's probably fine to constrain its api for now.